marklogic-client-api/src/test/resources/bootstrap.xqy

```diff
@@ -578,6 +578,8 @@ declare function bootstrap:security-config() {
         ) else (
             if ($user eq "valid") then (
                 bootstrap:security-eval('sec:create-user("valid", "valid unprivileged user", "x", (), (), (), ())')
+            ) else if ($user eq "rest-reader") then (
+                bootstrap:security-eval('sec:create-user("rest-reader", "rest-reader user", "x", ("rest-reader", "rest-reader-internal"), (), (), ())')
```

Comment:
In a customer adoption, an external user should never need a *-internal role.  

Instead, the security model is that code on the server should amp to the *-internal role for users that have the external role.

If the user for tests has the *-internal role, then the tests don't reliably indicate whether production users can execute the requests.